### ISSUE TITLE
Pagination cannot be used via keyboard only

### ISSUE THREAD

**Issue by kevfuzz** [mariuszfoltak/angular2-datatable#108]
Hi,

Any idea why you can't seem to tab into the pagination? For users with accessibility/mobility issues keyboard only navigation is vital.

Thanks,
Kevin.

**Comment by kevfuzz** [comment]
Should have also added that the column sorting has a similar issue.
